hooks/pre-commit.nocommit

```diff
@@ -11,11 +11,13 @@ fi
 # Redirect output to stderr.
 exec 1>&2
 
+DEFAULT_NO_COMMIT_RE='nocommit|testing-only'
```

Comment:
just to make sure we don't push it up. it's not really needed tho because CI will blow up if it sees `testing-only` anyway.